pom.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.dockstore</groupId>
+    <artifactId>dockstore-file-gs-plugin</artifactId>
+    <packaging>jar</packaging>
+    <name>dockstore-file-gs-plugin</name>
+    <version>0.0.1-SNAPSHOT</version>
+
+    <properties>
+        <plugin.id>${project.artifactId}</plugin.id>
+        <plugin.class>io.dockstore.provision.GSPlugin</plugin.class>
+        <plugin.version>${project.version}</plugin.version>
+        <plugin.provider>Walt Shands</plugin.provider>
+        <plugin.dependencies />
+        <github.url>scm:git:git@github.com:dockstore/gs-plugin.git</github.url>
+    </properties>
+
+    <distributionManagement>
```

Comment:
I don't think you need this section, since we will be doing GitHub releases, which is different than how s3 plugin is done.

Reply:
The distribution management section?

Reply:
Yes, the whole distributionManagement section should not be necessary.

Reply:
Removed distributionManagment section